### PR TITLE
chore(file-uploader): update controls and remove next folder

### LINKED ISSUE #12224: Storybook controls: provide FileUploader playground, controls

### PR DESCRIPTION
**Pull request by aledavila**
Closes #12224

Add playground story and removes next files 

#### Testing / Reviewing

Make sure stories are working as intended

### CODE REVIEW

File: packages/react/src/components/FileUploader/index.js [hunk 1]
```diff
@@ -5,24 +5,9 @@
...
-  FileUploaderSkeleton,
-  FileUploaderButton,
-  FileUploaderDropContainer,
-  FileUploaderItem,
-};
-
-export default FeatureFlags.enabled('enable-v11-release')
-  ? createClassWrapper(FileUploader)
-  : FileUploader;
+export default from './Fileuploader';
```
Comment (abbeyhrt) [inline packages/react/src/components/FileUploader/index.js]: I think the typo here is causing the CI to fail
```suggestion
export default from './FileUploader';
```

**Review by abbeyhrt — COMMENTED** [review]
Made a note about a typo 🙂

File: packages/react/src/components/FileUploader/index.js (hunk 1 above)
Comment (aledavila) [reply packages/react/src/components/FileUploader/index.js]: @abbeyhrt thank you ! I was going crazy trying to figure it out lol

**Comment by tw15egan** [comment]
Getting some errors when trying to view the `Default` and `Playground` stories

File: packages/react/src/components/FileUploader/stories/drop-container.js [hunk 2]
```diff
@@ -5,18 +5,28 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import React, { useState, useCallback, useEffect } from 'react';
+import React, { useState, useCallback, useEffect, useRef } from 'react';
 import classnames from 'classnames';
 import FileUploaderItem from '../FileUploaderItem';
 import FileUploaderDropContainer from '../FileUploaderDropContainer';
 import FormItem from '../../FormItem';
-import uid from '../../../tools/uniqueId';
```
Comment (tw15egan) [inline packages/react/src/components/FileUploader/stories/drop-container.js:13]: Was there an issue with using the `uid` internal function like this? Just wondering why the internals were copied in here

Comment (aledavila) [reply packages/react/src/components/FileUploader/stories/drop-container.js:13]: I'm not sure. I only moved this file over from the next folder out into the root. Looks like this is the implementation done during the v11 update.

**Review by tw15egan — APPROVED** [review]

**Review by abbeyhrt — APPROVED** [review]

**Pull request merged**